src/mariner/index.ts

```diff
@@ -2,5 +2,6 @@ export { DependencyDetailsRetriever } from '../dependency-details-retriever';
 export { readConfigFile, Config } from '../config';
 export { Issue, IssueFinder } from '../issueFinder';
 export { Logger, getLogger, setLogger } from '../tab-level-logger';
-export { generateConfluenceMarkup as generateConfluenceMarkdown } from '../Utilities/generateConfluenceMarkup';
+export { generateConfluenceMarkup } from '../Utilities/generateConfluenceMarkup';
```

Comment:
is it intentional that this is a breaking change, since it renames the exported confluence output function? I am inclined to think that the breaking change should probably be a separate PR.

Reply:
yes it's intentional, since we are adding in another file, [generateGitHubMarkdown](https://github.com/indeedeng/Mariner/pull/111/files#diff-c7afdb73d2ada6c84b773c723c7d964fe259594aba2d6118c9272dc6ee993f30) and we also have [generateGitHubMarkup](https://github.com/indeedeng/Mariner/pull/111/files#diff-def8a2a22dd145c0da7302d9204fe4d8c9b233248707edaad534a845af6e0d33) it was renamed to markup instead of markdown